backend/src/Response/StoreOwner/StoreOwnerCompleteAccountStatusGetResponse.php

```diff
@@ -4,5 +4,5 @@
 
 class StoreOwnerCompleteAccountStatusGetResponse
 {
-    public string $completeAccountStatus;
+    public ?string $completeAccountStatus;
```

Comment:
not an ideal at all solution, but okay for now

Reply:
Expected that truly.